tnp/templates/consent/profile.html

```diff
@@ -0,0 +1,395 @@
+{% extends 'base.html' %}
```

Comment:
Thinking about moving this page to card-based UI as seen in `job.html` page. Tell me your opinion? @abhishek981996